cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -3350,15 +3382,43 @@ fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(
                     ctx.emit(match elem_ty {
                         types::F32 => Inst::xmm_mov(SseOpcode::Movss, RegMem::mem(amode), dst),
                         types::F64 => Inst::xmm_mov(SseOpcode::Movsd, RegMem::mem(amode), dst),
+                        types::I8X8 => {
+                            println!("8x8");
```

Comment:
Leftover debugging printlns (here and a few below)?

Reply:
@cfallin So sorry. I could I have sworn I removed those in the latest push. Ok, Thanks.

Reply:
@cfallin @abrown Also not sure what we are doing? Is it ok to merge this or do we need to wait for #2432? I did run this against the spec tests and passed no problem, but of course there is a intermittent spec test problem with certain SIMD float instructions that we need to resolve before turning those tests back on.

Reply:
@jlb6740 Thanks! IMHO, it might be better to merge this now, just in order to have one less outstanding PR to keep track of, and given that everything passes fine for you locally; when we eventually turn the tests back on, after finding the intermittent bug, we will know right away if this has somehow broken in the meantime.

Reply:
I guess we should merge #2480 and #2489 then? I'll rebase them and take them out of draft mode... they're just sitting there getting stale right now.